src/modules/transaction/hooks/useTransactionFee/useTransactionFee.js

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react';
+import { useCommandSchema } from '@network/hooks/useCommandsSchema';
+import { useAuth } from '@auth/hooks/queries/useAuth';
+import { computeTransactionMinFee, getParamsSchema } from './utils';
+/**
+ *
+ * @param {object} data
+ * @param {boolean} data.isValid Whether the transaction is valid or not. TxComposer defines this
+ * @param {string} data.senderAddress The sender address in Lisk 32 format
+ * @param {object} data.transaction Transaction object as Lisk Element expects without fee
+ * @returns {object} The fee object with a total value, and a component value as an array of fees
+ * that contribute in the total value
+ */
+export const useTransactionFee = ({ isValid, senderAddress, transaction }) => {
+  const {
+    data: auth,
+    isLoading,
+    isFetched,
+  } = useAuth({ config: { params: { address: senderAddress } } });
```

Comment:
Maybe we should add an enabled option for these hooks which checks for `isValid` ?